data/assets/scene/solarsystem/sun/EUV_layer.asset

```diff
@@ -0,0 +1,56 @@
+asset.require("spice/base")
+local assetHelper = asset.require("util/asset_helper")
+local propertyHelper = asset.require('../../../util/property_helper')
+local transforms = asset.require("./transforms")
+
+local textureSourcePath = asset.syncedResource({
+  Type = "HttpSynchronization",
+  Name = "euv_textures_bastille_event",
+  Identifier = "euv_textures_bastille_event",
+  Version = 1
+})
+
+local EUVLayer = {
+  Identifier = "EUV-Layer",
+  Parent = transforms.SunIAU.Identifier,
+  Renderable = {
+    Type = "RenderableTimeVaryingSphere",
+    Size = 6.96E8, -- Slightly bigger than the sun renderable,
+    Enabled = true,
+    TextureSource = textureSourcePath,
+    --BlendMode = "Additive",
```

Comment:
Can this commented-out line be removed? :) Commented-out lines in assets have a tendency of being confusing